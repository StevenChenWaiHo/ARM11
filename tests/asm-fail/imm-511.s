mov r1, #511