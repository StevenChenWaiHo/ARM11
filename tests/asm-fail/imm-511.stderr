tests/asm-fail/imm-511.s:1:13: `#511` out of range for immediate
